src/dispatch/cli.py

```diff
@@ -568,6 +568,10 @@ def dump_database():
         _env={"PGPASSWORD": password},
     )
 
+@dispatch_database.command("load_sample_data")
+def load_sample_data():
+    """Load sample data into the database."""
+    pass
```

Comment:
Should we reuse the `restore` command/function with a different path?

Reply:
I made changes to re-use the `restore` command. My sample data is still not working and I think I'll have to make some edits to it. I'll ping you here once I have something working. Thanks for the feedback!